arviz/data/io_pymc3.py

```diff
@@ -89,7 +89,8 @@ def __init__(
         # this permits us to get the model from command-line argument or from with model:
         try:
             self.model = self.pymc3.modelcontext(model or self.model)
-        except TypeError:
+        except TypeError as e:
+            _log.error("Got error %s trying to find log_likelihood in translation.", e)
```

Comment:
Kind of a side note, we may want to port that to https://github.com/pymc-devs/pymc3/blob/v4/pymc3/backends/arviz.py, but I would wait for a while before that because log likelihood computation will probably be rewritten completely.

Reply:
I'm going to mark this as resolved, since it's for future reference.